bio2zarr/vcf_utils.py

```diff
@@ -396,6 +399,9 @@ class VcfIndexType(Enum):
 class IndexedVcf(contextlib.AbstractContextManager):
```

Comment:
The name is now a bit misleading since the VCF is not necessarily indexed. The class is used for partitioning, iterating over variants, and getting metadata about number of variants. I can't think of a better name though!

Reply:
Yes, good point. I've renamed to "VcfFile". It's good enough.